### PR TITLE
Move -Djava.lang.string.substring.nocopy out of compatible options

### LINKED ISSUE #501: -Djava.lang.string.substring.nocopy=true not a Hotspot compatible option
https://www.eclipse.org/openj9/docs/cmdline_migration/

Just realized that `-Djava.lang.string.substring.nocopy=true` doesn't belong in the table of compatible options. It's an option that should be set for Hotspot compatibility. We'll need to add another table for this.

### PR DESCRIPTION
**Pull request by pshipton**
Fixes #501

Signed-off-by: Peter Shipton <Peter_Shipton@ca.ibm.com>

### CODE REVIEW

**Comment by pmhayward** [comment]
Jenkins doc stage

File: docs/openj9_newuser.md [hunk 1]
```diff
@@ -31,6 +31,12 @@ The Eclipse OpenJ9 virtual machine (VM) implements the [Java Virtual Machine Spe
 
 Although OpenJ9 implements its own command-line interface, many HotSpot options are recognized and accepted by the VM for compatibility. Any `-XX:` options that are not recognized by the VM are ignored by default, which prevents an application failing to start. You can turn off this behavior with the [-XX:-IgnoreUnrecognizedXXColonOptions](xxignoreunrecognizedxxcolonoptions.md) option.
 
+Set the following options for compatibility with Hotspot.
```
Comment (pmhayward) [inline docs/openj9_newuser.md]: After some discussion, the ID team decided that this this new table should preferably live in the "Switching to OpenJ9" (`cmdline_migration.md`) topic, at the end in a new section, something like this:
**Creating compatible behavior**
You can set the following options to make OpenJ9 behave in the same way as Hotspot.

Looking at that topic, I wondered if the `-XX:+UseNUMA` option currently in the "Equivalent options" table might arguably belong in this new table - isn't it an option you have to set to get the same default behavior as Hotspot?

Comment (pshipton) [reply docs/openj9_newuser.md]: I've moved the table and added -Xnuma:none to the list.

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by pmhayward** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by SueChaplain** [comment]
Jenkins doc stage

**Comment by pmhayward** [comment]
Jenkins doc stage

File: docs/openj9_newuser.md (hunk 1 above)
Comment (pmhayward) [reply docs/openj9_newuser.md]: Thanks - all looks good

**Pull request merged**